runtime/rasdump/dmpagent.c

```diff
@@ -886,7 +886,12 @@ static omr_error_t
 doJavaDump(J9RASdumpAgent *agent, char *label, J9RASdumpContext *context)
 {
 	J9JavaVM *vm = context->javaVM;
-	if ((0 != strcmp(label, "/STDOUT/")) && (0 != strcmp(label, "/STDERR/"))) {
+
+	if ((0 == strcmp("-", label)) || (0 == j9_cmdla_stricmp(label, J9RAS_STDOUT_NAME))) {
+		strcpy(label, J9RAS_STDOUT_NAME);
+	} else if (0 == strcasecmp(label, J9RAS_STDERR_NAME)) {
```

Comment:
All uses of `strcasecmp`, including this one, will need to switch to `j9_cmdla_stricmp`.

Reply:
Sorry I forgot about this one

Reply:
@AidanHa Will you push your changes shortly?

Reply:
Sorry, I pushed the changes yesterday morning, but they didn't show up until i did a rebase...